_data/harnesses/creating-first-apache-kafka-streams-application/kstreams.yml

```
@@ -0,0 +1,157 @@
+
+        - action: execute
+          file: tutorial-steps/dev/gradle-wrapper.sh
+          render:
+            file: tutorials/creating-first-apache-kafka-streams-application/kstreams/markup/dev/make-gradle-wrapper.adoc
+
+        - action: execute
+          file: tutorial-steps/dev/make-configuration-dir.sh
+          render:
+            file: tutorials/creating-first-apache-kafka-streams-application/kstreams/markup/dev/make-config-dir.adoc
+
+        - action: make_file
+          file: configuration/dev.properties
+          render:
+            file: tutorials/creating-first-apache-kafka-streams-application/kstreams/markup/dev/make-dev-file.adoc
+            
+    - title: Create a Utility class 
+      content:
+        - action: execute
+          file: tutorial-steps/dev/make-src-dir.sh
+          render:
+            file: tutorials/creating-first-apache-kafka-streams-application/kstreams/markup/dev/make-src-dir.adoc
+
+        - action: make_file
+          file: src/main/java/io/confluent/developer/Util.java
+          render:
+            file: tutorials/creating-first-apache-kafka-streams-application/kstreams/markup/dev/make-util.adoc 
+
+    - title: Create the Kafka Streams topology
+      content:
+        - action: make_file
+          file: src/main/java/io/confluent/developer/KafkaStreamsApplication.java
+          render:
+            file: tutorials/creating-first-apache-kafka-streams-application/kstreams/markup/dev/make-topology.adoc
+
+    - title: Compile and run the Kafka Streams program
+      content:
+        - action: execute
+          file: tutorial-steps/dev/build-uberjar.sh
+          render:
+            file: tutorials/creating-first-apache-kafka-streams-application/kstreams/markup/dev/build-uberjar.adoc
+
+        - action: execute_async
+          file: tutorial-steps/dev/run-dev-app.sh
+          render:
+            file: tutorials/creating-first-apache-kafka-streams-application/kstreams/markup/dev/run-dev-app.adoc
+
+    - title: Stream events using a console consumer
+      content:
+        - action: skip
+          render:
+            file: tutorials/creating-first-apache-kafka-streams-application/kstreams/markup/dev/run-consumer.adoc
+
+test:
+  steps:
+    - title: Create a test configuration file
+      content:
+        - action: make_file
+          file: configuration/test.properties
+          render:
+            file: tutorials/creating-first-apache-kafka-streams-application/kstreams/markup/test/make-test-file.adoc
+
+    - title: Write a test
+      content:
+        - action: execute
+          file: tutorial-steps/test/make-test-dir.sh
+          render:
+            file: tutorials/creating-first-apache-kafka-streams-application/kstreams/markup/test/make-test-dir.adoc
+
+        - action: make_file
+          file: src/test/java/io/confluent/developer/DynamicOutputTopicTest.java
```

```suggestion
          file: src/test/java/io/confluent/developer/KafkaStreamsApplication.java
```
Sorry I missed this in the last review pass.